### PR TITLE
[Scene] [User] Update add report ver2

### PR DESCRIPTION
**Pull request by antoniusfilian**
update dropdown, add submit button, update add-report layout and logic

image
image
image
image
image
image

### CODE REVIEW

File: src/app/core-ui/custom-text-view/custom-text-view.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (danielindracahyadi) [inline src/app/core-ui/custom-text-view/custom-text-view.component.spec.ts]: why there is still custom-text-view here

File: src/app/components/card/card.component.ts [hunk 2]
```diff
@@ -13,7 +13,6 @@ export class CardComponent implements OnInit {
 
   ngOnInit() {
     const asd = 'lasglsalasl';
-    console.log(this.data[0]);
 
 
     // let a,b,c,d,e;
```
Comment (danielindracahyadi) [inline src/app/components/card/card.component.ts]: please remove this comment

File: src/app/core-ui/custom-text-view/custom-text-view.component.spec.ts (hunk 1 above)
Comment (antoniusfilian) [reply src/app/core-ui/custom-text-view/custom-text-view.component.spec.ts]: dunno

File: src/app/components/card/card.component.ts (hunk 2 above)
Comment (antoniusfilian) [reply src/app/components/card/card.component.ts]: ok

Comment (antoniusfilian) [reply src/app/components/card/card.component.ts]: in my project, there is no console.log. Why did it happen?

image

**Pull request merged**